src/extension/content-script/batteries/YouTubeVideo.ts

```diff
@@ -62,6 +88,40 @@ const battery = async (): Promise<void> => {
   ]);
 };
 
+const battery = async (): Promise<void> => {
+  function waitForDescription() {
+    // to observe changes on page change
+    const description = document.querySelector(
+      "div#bottom-row.style-scope.ytd-watch-metadata"
+    );
+    // because the above element doesn't have complete text
+    const descriptionContent = document.querySelector(
+      "#columns #primary #primary-inner #meta-contents #description .content"
+    );
+    // we need to ensure both are present because they don't load at the same time
+    // and this causes an issue because "description" alone isn't full
+    if (description && descriptionContent) {
+      clearInterval(descriptionInterval); // Stop checking for the element
+
+      if (!window.ALBY_BATTERY) {
```

Comment:
what is ALBY_BATTERY for?

Reply:
It just adds it for the session, i.e. if refreshed it goes away

> what if two different batteries used the same ALBY_BATTERY variable

No, because it adds only in the youtube.com/video links; if there's any other battery which does this then that too would limited to the regex pattern of that battery